src/value-converters/sanitize-html-html-sanitizer.ts

```diff
@@ -0,0 +1,46 @@
+import sanitize from 'sanitize-html'
+import {HTMLSanitizer} from "aurelia-templating-resources"
+
+/**
+ *  Sanitizes HTML tags for use in `innerhtml` attribute.
+ *  <p>You have to configure Aurelia DI to use an implementation on configuration stage.</p>
+ *  <p>Usage: <code><span innerhtml.bind="insecureValue|sanitizeHTML"></span></code></p>
+ * @author Simon Beyer <Simon.Beyer@t-systems.com>
+ */
+export class SanitizeHtmlHtmlSanitizer implements HTMLSanitizer {
+    private _options
+
+    constructor() {
+        this.restoreDefaults()
+    }
+
+    get options(): sanitize.IOptions {
+        return this._options
+    }
+
+    restoreDefaults(): SanitizeHtmlHtmlSanitizer {
+        this._options = { ...sanitize.defaults }
+        return this
+    }
+
+    /** A sensible default apart from {@link sanitize}s own: Escape everything, auto-correct nothing. <p>Exceptions may be added as desire is discovered.</p> */
+    escapeAllTheThings(): SanitizeHtmlHtmlSanitizer {
```

Comment:
I dont like the method's name, because it's to unspecific. It would be better to call it `escapeEverything` or `setParanoidOptions`.

```typescript
sanitizer.setParanoidOptions();
```

Reply:
Mmh. "all the things" and "everything" are pretty equivalent expressions, are they not? Different language levels.
But I like "Paranoid Options" better than both of them, so I'll go with that one.